src/routes/view-content/data-fetchers.ts

```diff
@@ -198,10 +198,7 @@ async function getAdditionalResourcesForPassage(
 function updateAndGetPreferredIds(bibleIdsInCurrentLanguage: number[]) {
     const defaultCurrentLanguageBibleId = bibleIdsInCurrentLanguage[0];
     let preferredIds = get(preferredBibleIds);
-    if (
-        defaultCurrentLanguageBibleId &&
-        preferredIds.every((existingId) => !bibleIdsInCurrentLanguage.includes(existingId))
-    ) {
```

Comment:
This logic was making it so that the current language's Bible was always selected. That was fine in the old paradigm but felt awkward with the new Bible selection menu.